api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/eligibility/EligibilityAndEntitlement.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.dhsc.htbhf.claimant.model.eligibility;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.dhsc.htbhf.claimant.entitlement.PaymentCycleVoucherEntitlement;
+import uk.gov.dhsc.htbhf.eligibility.model.EligibilityStatus;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class EligibilityAndEntitlement {
```

Comment:
I think there may be a better name here, something to do with the `PaymentCycle`, how about `PaymentCycleCalculation`, `PaymentCycleDetermination` or `PaymentCycleEvaluation` or something similar, I never like a class name with `And` in it.